spec/assessment_spec.rb

```diff
@@ -47,7 +47,7 @@
       expect(@response.some_odd([2, 3, 4, 6])).to eq(true)
     end
     it 'returns true when passed 2..6' do
-      expect(@response.every_even(2..6)).to eq(true)
+      expect(@response.some_odd(2..6)).to eq(true)
```

Comment:
Huh. Were these errors in the tests? If so, great job catching them! Can you please add an issue to this assessment with your changes to the tests?